src/components/checkbox/Checkbox.props.ts

```diff
@@ -0,0 +1,33 @@
+import { TCheckboxSizeToken } from './Checkbox.types';
+
+type PCheckbox = {
+    /**
+     * whether the checkbox is rendered with a label
+     * @default 'Public'
+     * */
+    label?: string;
+    /**
+     * the size of the checkbox
+     * @default 16
+     * */
+    size?: TCheckboxSizeToken;
+    /**
+     * whether the checkbox has an error
+     * @default false
+     * */
+    hasError?: boolean;
+    /**
+     * whether the checkbox is checked
+     * @default false
+     * */
+    checked?: boolean;
+    /**
+     * whether the checkbox is disabled for interaction
+     * @default false
+     * */
+    disabled?: boolean;
+    className?: string;
+    onClick?: () => void;
```

Comment:
This should have a description. Otherwise it will not be extracted from `react-docgen-typescript`